typings.json

```diff
@@ -3,6 +3,7 @@
     "core-decorators": "github:retyped/core-decorators-tsd-ambient/core-decorators.d.ts",
     "core-js": "registry:dt/core-js#0.0.0+20160317120654",
     "fetch.js": "github:ryan-codingintrigue/typescript-fetch/fetch.js.d.ts",
+    "node": "registry:dt/node#6.0.0+20160909174046",
```

Comment:
that was just an experiment to make this typings.json look similar to another project of mine